lib/Runtime/Base/WindowsGlobalizationAdapter.h

```diff
@@ -142,6 +156,24 @@ namespace Js
         static HRESULT VerifyResult(HSTRING * result, HRESULT errCode);
 #endif
     };
-}
-#endif
+#endif // INTL_WINGLOB
+
+#ifdef INTL_ICU
```

Comment:
That might be a good idea. But, since this is conditional compilation anyway and calling convention is going to be different (as informed by interfaces to the libraries), it might not be worth the extra overhead that polymorphism would add.

I'll consider it.

---
In reply to: [124413902](https://github.com/Microsoft/ChakraCore/pull/3214#discussion_r124413902) [](ancestors = 124413902)